examples/standalone/library-import/README.md

```diff
@@ -0,0 +1,37 @@
+# Standalone Analyzer Library Import Example
```

Comment:
In the `examples/*` directory, I have 2 example implementations:
1. `cli` - build and call a simple CLI (the toy build from Abhi's example [here](https://github.com/codecov/codecov-javascript-bundler-plugins/pull/71)!)
2. `library-import` - build a simple script that calls our new `standalone-analyzer` library


This is the start of `library-import` example